posthog/models/feature_flag.py

```diff
@@ -110,14 +129,119 @@ def get_filters(self):
                 ]
             }
 
+    def transform_cohort_filters_for_easy_evaluation(self):
```

Comment:
Food for thought: could this condition conversion be useful elsewhere to warrant this to be on the cohort object rather than a util on feature flag?

Reply:
A part of it may be useful, but the constraints here seemed too specific to feature flags to me:

Things like disregarding further nested cohorts, cohorts AND'ed with another ff filter, etc. etc.

Insights can handle a lot more cases, and here we're relying on feature flag groups to infer the shape (which look different from general insight filters) - that it didn't make sense to me to keep this util on a cohort instead.